dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dataset/deaccession/DatasetDeaccessionServiceTest.java

```diff
@@ -18,25 +15,27 @@
 
 import java.sql.Timestamp;
 import java.time.LocalDateTime;
+import java.util.Collections;
 
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 @MockitoSettings(strictness = Strictness.LENIENT)
 public class DatasetDeaccessionServiceTest {
```

Comment:
I don't know if we really need this test now – I was able to refactor it easily, because of the architecture of the service, but this is only a coincidence.

Reply:
We could leave it. IMO this test doesn't really check everything that it should (missing verify of the actual arguments - now it only verifies that something was executed but it doesn't check what was the arguments).

Reply:
After restructuring DeaccessionService I found that it is not possible to retain this unit test without making completely new set of tests with different way of testing, so I've decided to remove it (we have integration test so I think it's sufficient).